content/applications/hr/referrals/alerts.rst

```diff
@@ -0,0 +1,56 @@
+======
+Alerts
+======
+
+In the *Referrals* application, it is possible to post a message, also referred to as an *alert*, at
+the top of the dashboard to share important information with users.
+
+Alerts appear on the main dashboard for the specified time configured on the individual alert.
+
+.. image:: alerts/alerts.png
+   :align: center
+   :alt: Two alert banners appear above the user's photo.
+
+Create an alert
+===============
+
+Only users with :guilabel:`Administrator` rights for the *Recruitment* application can create
```

Comment:
```suggestion
Only users with *Administrator* access rights for the *Recruitment* application can create
```
italics should be used since the element being mentioned is being shown/seen on the screen at this time.